scss/alerts.scss

```diff
@@ -0,0 +1,209 @@
+.alert {
+  position: relative;
+  padding: 1rem 1rem;
+  border: 3px solid transparent;
+  border-radius: 0.25rem;
+  animation-fill-mode: forwards !important;
+  animation-delay: 2s !important;
+  transition: border-width 0.6s linear;
+  a {
+      font-weight: 700;
+      position: relative;
+      text-decoration: none;
+    }
+}
```

Comment:
this might work, but i still think it would be better to wrap the whole definitions in one .alert{} block ;)